include/yaramod/types/hex_string.h

```diff
@@ -192,15 +201,15 @@ class HexStringJump : public HexStringUnit
 		// If both low and high bound is defined and they are the same, it is the fixed jump.
 		if (_low.has_value() && _high.has_value() && _low.value() == _high.value())
 		{
-			ss << _low.value();
+			ss << _low.value()->getUInt64_t();
```

Comment:
I have removed `isString` and similar methods from `Literal` class and replace these with single templated `Literal::is` function. Similarly, the templated `Literal::get()` has been introduced.

But I would prefer to keep the in the methods `getString`, 'getInt', etc. of `Token` class because it is more readable. Would that be alright?

Reply:
Sure but why keeping `_t` suffix there. It's the internal name of C++ type `std::uint64_t`. At least call it `getUInt64` instead.

Reply:
That's a good point! I renamed the `getInt64_t`, `getUInt64_t`, `isInt64_t` and `isUInt64_t` and removed the `_t`.